### PR TITLE
Movie details page api consumption

### PR DESCRIPTION
**Pull request by NSMcEnroe**
## Description
This PR will flush out the movies show page.

-Includes: -Button to create new viewing parties
-button to return to discover page
-updates to movie PORO
-new review PORO
-Added new facade methods and service methods

-Unfortunately, we probably wont have time to add additional tests for facades, services, edge cases, and sad paths.

### Type of Change

- [x] **fix**
- [x] **feat**
- [x] **test**
- [x] **refactor**
- [ ] **docs**

### Checklist

- [x] **Code has been self reviewed**
- [x] **Code runs without any errors**
- [ ] **Thorough testing has been implemented if adding feature**
- [x] **All tests pass**

### CODE REVIEW

File: app/poros/movie.rb [hunk 1]
```diff
@@ -1,9 +1,30 @@
...
+      rest = @runtime_minutes % 60
+      "#{hours}:#{rest}" 
+  end
+
+  def all_genres
+    genre_names = @genres.map do |genre|
+      genre[:name]
+    end
+    genre_names.join(', ')
   end
```
Comment (ttakahashi1591) [inline app/poros/movie.rb:29]: Great use of these methods to reformat the data we are pulling from the API to ensure it is presented in the way that we need it.

File: app/views/movies/show.html.erb [hunk 2]
```diff
@@ -1 +1,29 @@
...
+
+<h1><%= "#{@movie.show_movie.original_title}" %> Details</h1>
+
+<ul class="movie-details">
+  <li><%= @movie.show_movie.original_title %></li>
+  <li><%= @movie.show_movie.vote_average %></li>
+  <li><%= @movie.show_movie.runtime %></li>
+  <li><%= @movie.show_movie.all_genres %></li>
+  <li><%= @movie.show_movie.summary %></li>
+</ul>
```
Comment (ttakahashi1591) [inline app/views/movies/show.html.erb:13]: I see that we are using those methods that we built in our poros here so that it will show up on the webpage correctly.

File: spec/features/movies/show_spec.rb [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    visit "/users/#{user.id}/movies/238"
+
+    expect(page).to have_content("Total Reviews: 5")
+    expect(page).to have_content("futuretv")
+    expect(page).to have_content("They’re both flawless crime dramas")
+    expect(page).to have_content("Rating: 10.0")
+  end
+
+  it "displays 0 reviews if no one has given the review", :vcr do
```
Comment (ttakahashi1591) [inline spec/features/movies/show_spec.rb:51]: I like that you broke down the testing based off our how we are retrieving the information form the API and if there is any manipulation necessary after consuming.
